types/debug/index.d.ts

```diff
@@ -42,6 +42,7 @@ declare namespace debug {
         enabled: boolean;
         log: (...args: any[]) => any;
         namespace: string;
+        destroy: () => void;
```

Comment:
From the [library's source](https://unpkg.com/debug@4.1.1/dist/debug.js), it appears that this actually returns `boolean`, not void.